FirebaseRemoteConfig/Sources/RCNConfigRealtime.m

```diff
@@ -458,7 +464,15 @@ - (void)URLSession:(NSURLSession *)session
                                                              error:&dataError];
   NSString *targetTemplateVersion = _configFetch.templateVersionNumber;
   if (dataError == nil) {
-    targetTemplateVersion = [response objectForKey:kTemplateVersionNumberKey];
+    if ([response objectForKey:kTemplateVersionNumberKey]) {
+      targetTemplateVersion = [response objectForKey:kTemplateVersionNumberKey];
+    }
+    if ([response objectForKey:kIsFeatureDisabled]) {
```

Comment:
is there a scenario where these fields will not be populated?

Reply:
In theory there shouldn't be but the response may be mangled in some way. So added checks just in case.